public/data/object/components.json

```diff
@@ -1,35 +1 @@
-[
-  {
-    "elements": [
-      {
-        "amount": 0.734063,
-        "material": "59b42284-3e45-5343-8a20-1d7d66137461"
-      }
-    ],
-    "id": "ad9d7f23-076b-49c5-93a4-ee1cd7b53973",
-    "name": "Dossier plastique (PP)",
-    "scopes": ["object", "veli"]
-  },
-  {
-    "elements": [
-      {
-        "amount": 0.91125,
-        "material": "59b42284-3e45-5343-8a20-1d7d66137461"
-      }
-    ],
-    "id": "eda5dd7e-52e4-450f-8658-1876efc62bd6",
-    "name": "Assise plastique (PP)",
-    "scopes": ["object", "veli"]
-  },
-  {
-    "elements": [
-      {
-        "amount": 35,
-        "material": "59b42284-3e45-5343-8a20-1d7d66137461"
-      }
-    ],
-    "id": "190276e9-5b90-42d6-8fbd-bc7ddfd4c960",
-    "name": "Cadre plastique",
-    "scopes": ["object", "veli"]
-  }
-]
+[]
```

Comment:
Please note that at this point, we don't have any default components for the Object domain. Dunno if it's an issue but this feels very strange to me. cc @camcoq-start @paulboosz